src/main/java/com/redislabs/modules/rejson/JReJSON.java

```diff
@@ -122,7 +122,7 @@ private static void assertReplyNotError(final String str) {
      * Helper to check for an OK reply
      * @param str the reply string to "scrutinize"
      */
-    private static void assertReplyOK(final String str) {
+    protected static void assertReplyOK(final String str) {
```

Comment:
We can trust a few private methods.
```suggestion
    private static void assertReplyOK(final String str) {
```

Reply:
Making them private prevents me from testing them. I just make them package level instead.

Reply:
Skip testing this.